python/dask_cudf/dask_cudf/tests/test_sort.py

```diff
@@ -53,23 +52,21 @@ def test_sort_repartition():
 
 
 @pytest.mark.parametrize("by", ["a", "b", ["a", "b"]])
-def test_sort_values_with_nulls(by):
-    df = cudf.DataFrame(
+@pytest.mark.parametrize(
+    "data",
+    [
         {
             "a": list(range(50)) + [None] * 50 + list(range(50, 100)),
             "b": [None] * 100 + list(range(100, 150)),
-        }
-    )
-    ddf = dd.from_pandas(df, npartitions=10)
-
-    # assert that quantile divisions of dataframe contains nulls
-    divisions = quantile_divisions(ddf, by, ddf.npartitions)
-    if isinstance(divisions, list):
-        assert None in divisions
-    else:
-        assert all([divisions[col].has_nulls for col in by])
+        },
+        {"a": list(range(15)) + [None] * 5, "b": list(reversed(range(20)))},
+    ],
+)
+def test_sort_values_with_nulls(data, by):
+    df = cudf.DataFrame(data)
+    ddf = dd.from_pandas(df, npartitions=5)
 
     got = ddf.sort_values(by=by)
     expect = df.sort_values(by=by)
 
-    dd.assert_eq(got, expect)
+    dd.assert_eq(got, expect, check_index=False)
```

Comment:
This seems reasonable to me. If you also want to check the output index order, you could convert it to a column: `dd.assert_eq(got.reset_index(), expect.reset_index(), check_index=False)`